packages/smoke-test/test/commands/fix.test.ts

```diff
@@ -47,7 +47,7 @@ describe('fix command', () => {
     expectFile(join(projectRoot, 'src/app.ts')).toMatchSnapshot();
   });
 
-  test('ember_js_app_4.11', async () => {
+  test('ember_js_app_4.11 --mode=single-pass', async () => {
```

Comment:
The `drain` is default mode now, need to test opposite - `single-pass`.

Reply:
🤣 My question above is answered by this. 👍